plaidml2/bridge/keras/__init__.py

```diff
@@ -1060,16 +1060,24 @@ def not_equal(lhs, rhs):
 
 @_log_call
 def normalize_batch_in_training(x, gamma, beta, reduction_axes, epsilon=1e-3):
-    I = x.tensor
-    ndims = I.shape.ndims
-    if reduction_axes == None:
-        raw_axes = [ndims - 1]
+    ndims = ndims(x)
```

Comment:
Personally I slightly prefer simplicity of code to gaining a bit of perf here, but this could also be
```
shape = shape(x)
ndims = len(shape)
```
to save us one call through the FFI. @flaub thoughts?
(also, if we leave this as-is, I think the Keras function name is `ndim` not `ndims`)